src/components/Autocomplete/AutocompleteBase/AutocompleteBase.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import AutocompleteBase, { AutocompleteBaseProps } from ".";
+import TextField from "../../Textfields/TextFieldBase";
+
+import { Story } from "@storybook/react";
+
+const options = [
+    { title: "Thien Duc" },
+    { title: "Kien Tran" },
+    { title: "Binh Pham" },
+    { title: "Thanh Binh" },
+];
+
+export default {
+    title: "Components/AutocompleteBase",
+    component: AutocompleteBase,
+};
+
+const Template: Story<AutocompleteBaseProps<Object, boolean, boolean, boolean>> = (args) => {
+    return <AutocompleteBase {...args} />;
+};
+export const Basic = Template.bind({});
+Basic.args = {
+    options: options,
+    getOptionLabel: (option: any) => option.title,
```

Comment:
This is why you need generic. If you do it right, the option will automatically get its type 
```suggestion
    getOptionLabel: (option) => option.title,
```